readme.txt

```diff
@@ -35,32 +35,47 @@ Upon successful activation, **SparkPost** will appear on the _Settings_ menu in
 
 Fill in each plugin configuration field:
 
-* **enable?**: Check this box to enable your plugin :)
-* **from name**: a human-friendly name to show in 'From' headers
-  * e.g. your name or your site's name
+* Check the "Send email using SparkPost" box to enable the plugin
 
-* **from email**: your 'From' email address
-  * e.g. yourname@yourdomain.com
+* **Method**: select whether to send using the HTTP API or SMTP
 
-* **SMTP password**: A SparkPost API key with *Send via SMTP* permission
+* **API Key**: a SparkPost API key with *Send via SMTP* and *Transmission Read/Write* permissions
   * Hint: they look like this: 39fb780c182927cde6baddab00f67676feed1beef17
 
+* **Enable tracking**: select whether to enable open and click tracking
+
+* **Template**: SparkPost template to use when sending email
```

Comment:
I think we should just remove this part altogether about the fields. It's just duplicating what's on the settings page.